scratch-js/Project.mjs

```diff
@@ -156,28 +156,6 @@ export default class Project {
     return [...this.spritesAndClones, this.stage];
   }
 
-  playSound(url) {
-    return new Promise((resolve, reject) => {
-      const audio = new Audio(url);
-
-      const sound = { audio, hasStarted: false };
-
-      const soundEnd = () => {
-        this._stopSound(sound);
-        resolve();
-      };
-      audio.addEventListener("ended", soundEnd);
-      audio.addEventListener("pause", soundEnd);
-      audio.addEventListener("error", reject);
-
-      this.playingSounds.push(sound);
-
-      audio.play().then(() => {
-        sound.hasStarted = true;
-      });
-    });
-  }
-
   _stopSound(sound) {
```

Comment:
`_stopSound` can be safely deleted, right?

Reply:
Oh yup! Thanks. Pushed!